rtlib/bigint.hh

```diff
@@ -22,8 +22,8 @@
 }}} */
 
 
-#ifndef BIGINT_HH 
-#define BIGINT_HH
+#ifndef RTLIB_BIGINT_HH_
+#define RTLIB_BIGINT_HH_
```

Comment:
Is this a cpplint thing or our own new standard of naming constants?

Reply:
I wouldn't have touched it if not cpplint is forcing me to do. It is their style.